components/sections/footer/variant_b.tsx

```diff
@@ -2,150 +2,133 @@ import { urlFor } from "lib/sanity";
-                            />
-                          </svg>
-                        ) : (
-                          social?.socialMediaIcon?.image && (
-                            <Image
-                              src={urlFor(social?.socialMediaIcon?.image)}
-                              width={24}
-                              height={24}
-                              alt={
-                                social?.socialMediaIcon?.alt ??
-                                "footer-socialMedia-icon"
-                              }
-                            />
-                          )
-                        )}
-                      </a>
-                    )
-                )}
-              </div>
-            )}
-          </div>
-        </div>
-      </div>
+                        {links?.label}
+                      </Button>
+                    </li>
+                    {index + 1 !== length ? (
+                      <li className="hidden md:block">
+                        <svg
+                          className="w-4 h-4 mx-4 text-gray-500"
+                          xmlns="http://www.w3.org/2000/svg"
+                          fill="none"
+                          viewBox="0 0 24 24"
+                          stroke="currentColor"
+                        >
+                          <path
+                            strokeLinecap="round"
+                            strokeLinejoin="round"
+                            strokeWidth="2"
+                            d="M12 5v.01M12 12v.01M12 19v.01M12 6a1 1 0 110-2 1 1 0 010 2zm0 7a1 1 0 110-2 1 1 0 010 2zm0 7a1 1 0 110-2 1 1 0 010 2z"
+                          ></path>
+                        </svg>
+                      </li>
+                    ) : null}
+                  </React.Fragment>
+                ))}
+              </Flex>
+            </div>
+          )}
+        </Flex>
+        <Flex wrap align="center" justify="between" className="mx-20 mt-8">
+          <Text muted className="order-last text-sm ">
```

Comment:
Kindly conditionally render copyright element to only display if it is defined/has value.